plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -49,33 +46,64 @@ public static DistributedQueryRunner createIcebergQueryRunner()
         return createIcebergQueryRunner(
                 Map.of(),
                 Map.of(),
-                List.of());
+                Optional.empty());
     }
 
     public static DistributedQueryRunner createIcebergQueryRunner(
             Map<String, String> extraProperties,
             Map<String, String> connectorProperties,
-            Iterable<TpchTable<?>> tables)
+            Optional<File> metastoreDirectory)
```

Comment:
i would probably prefer overload `extraProperties, connectorProperties, tables`.
are we setting `metastoreDirectory` so often?

Reply:
Added overload method with `extraProperties, connectorProperties, tables`